src/ROOT-Sim.h

```diff
@@ -20,8 +20,12 @@
 #include <stdint.h>
 #include <stdlib.h>
 
-#define INIT	0
-#define DEINIT	UINT_MAX
+enum rootsim_event {
```

Comment:
Not to break compatibility (and keep the interface "easier"): Let's map LP_INIT to 0 and other events to higher values (like as we did with control messages in v2).

Reply:
I agree in mapping event values to higher numbers so that we can leave "easier" values to the model developer.
But, if we simply map LP_INIT to 0, we still break compatibility since it was still renamed from INIT. So, in my opinion, we either rename LP_INIT to INIT and map it to 0, or we move it up together with the other values.

Reply:
I forgot it's only a static library now!
Let's keep LP_INIT and move it upwards. We're currently moving up the major number, so there should be no problem.